chanzuckerberg/idseq-web
# Warn about unsaved changes

# Description

Following @happyimadesignr's request, this adds a standard warning about unsaved changes to the heatmap page. 

![image](https://user-images.githubusercontent.com/28797/55052454-4514bb80-5016-11e9-88b8-2257e4db007a.png)

It's gated on being `admin` like the save button. 

@happyimadesignr did you expect this to prompt on the first save only or all unsaved changes? As it stands, it works on all unsaved changes.

# Test

1. open a heatmap
2. click back, see no warning
3. open a heatmap
4. change taxon level
5. click back, see warning
6. open a heatmap
7. change taxon level
8. save
9. click back, see no warning
10. create a new heatmap
11. click back, see no warning
